spec.html

```diff
@@ -25247,7 +25247,10 @@ <h1>Static Semantics: ReferencedBindings</h1>
         <emu-alg>
           1. Return a List containing the first |IdentifierName|.
         </emu-alg>
-        <emu-grammar>ExportSpecifier : ModuleExportName `as` ModuleExportName</emu-grammar>
+        <emu-grammar>
+          ExportSpecifier :
+            ModuleExportName
+            ModuleExportName `as` ModuleExportName</emu-grammar>
```

Comment:
This is missing a newline:
```suggestion
            ModuleExportName `as` ModuleExportName
          </emu-grammar>
```